apps/api/src/app/widgets/pipes/limit-pipe/limit-pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { ArgumentMetadata, BadRequestException, Injectable, PipeTransform } from '@nestjs/common';
+
+@Injectable()
+export class LimitPipe implements PipeTransform {
+  private readonly minInt: number;
+  private readonly maxInt: number;
+  private readonly isOptional: boolean = false;
+  constructor(minInt: number, maxInt: number, isOptional?: boolean) {
+    this.minInt = minInt;
+    this.maxInt = maxInt;
+    if (isOptional != null) {
+      this.isOptional = isOptional;
+    }
```

Comment:
if you will set the default value on the constructor argument you won't need to have this check...

Reply:
True